torchvision/models/feature_extraction.py

```diff
@@ -203,6 +203,7 @@ def _set_default_tracer_kwargs(original_tr_kwargs: Optional[Dict[str, Any]]) ->
 def get_graph_node_names(
     model: nn.Module,
     tracer_kwargs: Optional[Dict[str, Any]] = None,
+    concrete_args: Optional[Dict[str, Any]] = None,
```

Comment:
Unfortunately this parameter must be added as the last parameter, otherwise there's a (low) risk of breaking code that is using non-keyword parameters. This API should have been using keyword-only parameters to avoid that, but it didn't.